core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -789,16 +786,6 @@ class WorkspaceService(protected val ctx: RawlsRequestContext,
   private def isAzureMcWorkspace(maybeMcWorkspace: Option[WorkspaceDescription]): Boolean =
     maybeMcWorkspace.flatMap(mcWorkspace => Option(mcWorkspace.getAzureContext)).isDefined
 
-  // TODO - once workspace migration is complete and there are no more v1 workspaces or v1 billing projects, we can remove this https://broadworkbench.atlassian.net/browse/CA-1118
```

Comment:
We may need to come up with a plan for the remaining v1 workspaces (~13000) in prod, as this will break workspace deletion for those workspaces and leave billing projects with these workspaces undeleteable. Probably out of scope for this ticket, but just calling it out

Reply:
Let's huddle about this a bit, I want to get a plan in place before we break something here.